docs/docs/_index.md

```diff
@@ -26,10 +26,10 @@ See the [clients page](clients) for the full list.
 
 ## Using with other metrics tools
 
-In the [RedisTimeSeries](https://github.com/RedisTimeSeries) organization you can
-find projects that help you integrate RedisTimeSeries with other tools, including:
+In the [Time Series](https://github.com/RedisTimeSeries) organization you can
```

Comment:
This is a name of a GitHub organization. We are not changing it for now.

Reply:
I've fixed this in my local branch and I'll update the PR once a few more comments come in.